### DISCUSSION TITLE
Uncaught Error, tried to register two views with the same name AIRMap

### DISCUSSION THREAD

**Discussion by alanwong0922** [react-native-maps/react-native-maps#4695] [Q&A]
### Summary

I would got the Uncaught Error after install the plugin 
tried to register two views with the same name AIRMap 

### Reproducible sample code

```TSX
import React from 'react';
import {View, Text, TouchableOpacity, TextInput, Dimensions, Image, SafeAreaView, Linking} from 'react-native';
import {MapView} from 'react-native-maps';

const winWidth = Dimensions.get('screen').width;
const winHeight = Dimensions.get('screen').height;

class OfficeHours extends React.Component{
    constructor(props){
        super(props);
        this.state = {
            region: {
                latitude: 37.78825,
                longitude: -122.4324,
                latitudeDelta: 0.0922,
                longitudeDelta: 0.0421,
            }
        }
    }

    render(){
        return(
            <View>
                 <SafeAreaView>
                    {/* MAP */}
                    <MapView region={this.state.region} />
                </SafeAreaView>
            </View>
        ) 
    }
}

export default OfficeHours;
```

### Steps to reproduce

I installed the plugin "react-native-maps" with the version 1.6.0,

### Expected result

Show the map on the screen 

### Actual result

Uncaught Error 
tried to register two views with the same name AIRMap 

### React Native Maps Version

1.6.0

### What platforms are you seeing the problem on?

Android, iOS (Apple Maps), iOS (Google Maps)

### React Native Version

0.69.2

### What version of Expo are you using?

Not using Expo

### Device(s)

IPad Air 4th

**Comment by avijit-developer** [comment]
I am getting this issue. Can you please let me know how I can solve it?

**Comment by esegebart** [comment]
Also getting the same error message. Following.

**Comment by artur-skydiver** [comment]
I suddenly started getting this error on a build that had worked before.
What helped me was clearing all possible caches and reinstalling all packages from scratch.
[how-to-clear-react-native-cache](https://stackoverflow.com/questions/46878638/how-to-clear-react-native-cache)
